Header header
std_msgs/String leg_name

geometry_msgs/Point local_tip_position
geometry_msgs/Point poser_tip_positions
geometry_msgs/Point walker_tip_positions

std_msgs/Float64 stance_progress
std_msgs/Float64 swing_progress

std_msgs/Float64 tip_force
std_msgs/Float64 delta_z
std_msgs/Float64 virtual_stiffness